FAQ.md

```diff
@@ -102,3 +103,37 @@ SSLv3/TLSv1/TLSv1.1/TLSv1.2
 
 So if the server supports TLS1.2, the communication should be encrypted by TLS 1.2 (during the handshake the client will provide the list of accepted security protocols and the server will pick one, so this case the server would pick TLS 1.2)
 
+## How to use SSL client certificate authentication
```

Comment:
Hi @rhajek,

as the FAQ additions are unrelated to the streaming PR could you please move them to a seperate PR.

Reply:
Hi @majst01,

I moved the FAQ additions to PR #518.